persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/AggregateEventTag.scala

```diff
@@ -3,10 +3,9 @@
  */
 package com.lightbend.lagom.javadsl.persistence
 
-import scala.annotation.tailrec
-import akka.persistence.journal.WriteEventAdapter
-import akka.persistence.journal.WriteEventAdapter
-import akka.persistence.journal.Tagged
+import org.pcollections.{ PSequence, TreePVector }
+
+import scala.collection.JavaConverters._
 
 object AggregateEventTag {
```

Comment:
@patriknw Note the new API here, so you can still create a simple tag using `AggregateEventTag.of`, but if you want to shard, you can create a shard using the `shard` method, passing the event type (and custom base name if needed), the number of shards, and the entity ID, and you can get all the shards using the `shards` method.  The `selectShard` and `shardTag` are public methods that exist so that the user can generate their own tag names if they need to. I could imagine this being useful if someday they need to reshard and work directly on the database tables to do that.